.github/workflows/daily.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 148
148:                     Should -BeTrue -Because (Get-Content $splat.FilePath -Raw)
```